build/yaml/functionalTestsBotsBuild.yml

```diff
@@ -0,0 +1,233 @@
+steps:
```

Comment:
I am not clear on what this yml does, can you explain?

Reply:
updated to dotnetBotsUpdateDependencies-CI.yml